src/style.css

```
@@ -1 +1,71 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgb(0 0 0 / 0.1);
+}
+
+header .logo {
+  display: inline;
+  float: left;
+  width: 40px;
+  height: 40px;
+  margin-top: 10px;
+}
+
+.nav,
+.link__box {
+  display: flex;
+  justify-content: flex-end;
+  margin: 0;
+}
+
+.link__box {
+  padding-left: 0;
+  width: 100%;
+}
+
+.link__item {
+  display: inline-block;
```

do you really need `inline-block` in flex?
```suggestion
  display: block;
```

---

Yes, you're right!